### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jdw2465VEN**
Mostly copy edits, along with a few questions/suggestions.

### CODE REVIEW

File: examples/citrix_adc/README.md [hunk 1]
```diff
@@ -235,9 +236,9 @@ Once the files copied to the Citrix ADC, Ansible needs to create the http servic
       delegate_to: localhost
 ```
 
-### Creating Virtual server on Citrix ADC
+## Create a virtual server on Citrix ADC
```
Comment (jdw2465VEN) [inline examples/citrix_adc/README.md:239]: I forgot to add "**Step #:**" here...

File: examples/citrix_adc/README.md [hunk 2]
```diff
@@ -182,9 +179,13 @@ By adding the instructions below to the playbook, we indicate the actions the  p
 
 ```
 
-### Creating http services on Citrix ADC
+## Create a certificate-key pair on Citrix ADC
```
Comment (jdw2465VEN) [inline examples/citrix_adc/README.md:182]: I forgot to add "Step 3:" here...add if this is indeed a missing step...see inline comment

File: examples/citrix_adc/README.md [hunk 3]
```diff
@@ -182,9 +179,13 @@ By adding the instructions below to the playbook, we indicate the actions the  p
...
+
+**DW: this step is missing; if it's done as part of another step, remove it from the list of steps under Getting Started.**
+
+## Create HTTP back-end services on Citrix ADC
```
Comment (jdw2465VEN) [inline examples/citrix_adc/README.md:186]: I forgot to add "**Step #:**" here...

File: examples/citrix_adc/README.md [hunk 4]
```diff
@@ -271,19 +272,19 @@ Now that the http services have been create, Ansible has to create a virtual IP
 
 ## Executing the playbook
```
Comment (jdw2465VEN) [inline examples/citrix_adc/README.md:273]: If executing the playbook must be done to complete the example, I strongly suggest adding this to the Getting Started section as a step and adding "**Step #:**" here

Comment (ricrodriguezg) [reply examples/citrix_adc/README.md:273]: I agree, this can be added to the "Getting Started" section

File: examples/citrix_adc/README.md (hunk 3 above)
Comment (ricrodriguezg) [reply examples/citrix_adc/README.md:186]: This is not part of another step, it needs to be on the Getting Started Section

**Review by ricrodriguezg — CHANGES_REQUESTED** [review]
I agree with the changes you've performed, just added some comments on the ones that need to be modified

File: examples/citrix_adc/README.md (hunk 3 above)
Comment (jdw2465VEN) [reply examples/citrix_adc/README.md:186]: Just need @ricrodriguezg to add content under Step 3: Create a certificate-key pair on Citrix ADC. Please merge and then add it to doc_improvements and I'll review it there

**Comment by jdw2465VEN** [comment]
Closing this PR since I opted to make another branch, jdw2465VEN-patch-3

**Pull request closed without merging**